<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>601</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Franklin Gothic Demi Cond</family>
    </font>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="welcome_page">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>801</width>
       <height>601</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="aButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>540</y>
       <width>211</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>点   击  进   入</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="operator_page">
    <widget class="QPushButton" name="connect_btn">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>320</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe Hebrew</family>
      </font>
     </property>
     <property name="text">
      <string>连 接 小 车</string>
     </property>
    </widget>
    <widget class="QLabel" name="ip_label">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>180</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe Hebrew</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::PreventContextMenu</enum>
     </property>
     <property name="text">
      <string>IP:</string>
     </property>
    </widget>
    <widget class="QLabel" name="port_label">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>240</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe Hebrew</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>PORT:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ip_line_edit">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>170</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>ESRI Pipeline US 1</family>
      </font>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="port_line_edit">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>230</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="on_time_label">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>30</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Franklin Gothic Demi Cond</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="bButton">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>560</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe Hebrew</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>MODE</string>
     </property>
    </widget>
    <widget class="QLabel" name="signin_label">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>130</y>
       <width>441</width>
       <height>181</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="flower_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>801</width>
       <height>601</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>flower_label</zorder>
    <zorder>on_time_label</zorder>
    <zorder>bButton</zorder>
    <zorder>signin_label</zorder>
    <zorder>ip_label</zorder>
    <zorder>port_label</zorder>
    <zorder>ip_line_edit</zorder>
    <zorder>port_line_edit</zorder>
    <zorder>connect_btn</zorder>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="QStackedWidget" name="stackedWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>801</width>
       <height>591</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_2">
      <widget class="QPushButton" name="stop_btn">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>180</y>
         <width>221</width>
         <height>201</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="backward_btn">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>360</y>
         <width>71</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="left_btn">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>280</y>
         <width>71</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="forward_btn">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>190</y>
         <width>71</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="right_btn">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>270</y>
         <width>71</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="cButton">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>400</y>
         <width>161</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Adobe Hebrew</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ESC</string>
       </property>
      </widget>
      <widget class="QLabel" name="back_label">
       <property name="geometry">
        <rect>
         <x>-9</x>
         <y>-6</y>
         <width>801</width>
         <height>601</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <zorder>back_label</zorder>
      <zorder>stop_btn</zorder>
      <zorder>backward_btn</zorder>
      <zorder>left_btn</zorder>
      <zorder>forward_btn</zorder>
      <zorder>right_btn</zorder>
      <zorder>cButton</zorder>
     </widget>
     <widget class="QWidget" name="page_3"/>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
